add i18n for support (#775)
taptap/tds-docs: i18n/en/docusaurus-plugin-content-docs/current/sdk/tap-support/features.md

> # Tap Customer Service Introduction

Reviewer: 产品的英文名称需要确认一下。

> Tap Game Customer Service assists the game operations team in solving player problems faster and better.

Reviewer: ```suggestion
Tap Game Customer Service assists the game operations team in solving the problems encountered by players faster and better.
```

## Key Benefits

### Rich tools to improve customer service efficiency

Tap Game Customer Service consists of two core systems: Ticket and Knowledge Base, providing a set of freely combinable functional modules, including announcements, FAQs, forms, etc., to solve players' problems and requests.

> - Ticket built-in classification-based customer service automatic assignment function, you can also customize triggers to achieve a more complex assignment mechanism, the first time the Ticket will be assigned to the most appropriate customer service hands.

Reviewer: ```suggestion
- The ticket module comes with the category-based auto-assignment function. You can also customize triggers to achieve a more complex assignment mechanism that can quickly assign tickets to the most appropriate staff members.
```

> - Tickets support custom fields that can be filled in by the player via a form or actively reported by the developer, providing detailed context for customer service.

Reviewer: ```suggestion
- Tickets support custom fields that can be filled in by the player or the developer via a form, providing detailed context for the staff.
```

    
- Multiple notification methods are supported to ensure you don't miss any player responses.
    

### Multi-dimensional analytics to support operations

> Tap Game Customer Service is designed to help solve player problems, but it is also designed to help operations identify issues, refine and accelerate valuable content, and improve the game itself.

Reviewer: ```suggestion
Tap Game Customer Service is designed to help solve player problems, but it is also designed to help the operation team identify issues, refine and accelerate valuable content, and improve the game itself.
```

> - Tickets have flexible search and filtering capabilities and support aggregation and analysis by classification, field, and status. It also supports data export for further analysis with third-party tools.

Reviewer: ```suggestion
- The ticket module has flexible search and filtering capabilities and support aggregation and analysis by category, field, and status. It also supports data export for further analysis with third-party tools.
```

> - Knowledge base articles provide user feedback to see behind the scenes what content is effective and what is not solving the player's problem.

Reviewer: ```suggestion
- Users can leave feedback on knowledge base articles so that you can see behind the scenes what content is effective and what is not solving the player's problem.
```

> ### Simple and Powerful Management Features

Reviewer: 同一篇文章里的各个标题大小写风格最好保持统一。这篇文章里可以全都用 [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)。

Tap Game Customer Service provides an intuitive role-based permission control system that supports custom roles in addition to pre-built roles for more flexible needs.

> The system provides managers with detailed data to evaluate the quantity and quality of customer service work, including but not limited to the number of Tickets handled, the timeliness of responses to players, and player satisfaction.

Reviewer: ```suggestion
The system provides managers with detailed data to evaluate the quantity and quality of customer service work, including but not limited to the number of tickets handled, the timeliness of responses to players, and player satisfaction.
```

### User Experience Designed for Players

> As a customer service focused on the gaming community, Tap Game Customer Service provides an in-game customer service experience. Players don't have to switch apps to use the customer service feature, and they can receive in-game prompts immediately after customer service replies.

Reviewer: ```suggestion
As a customer service tool focused on gaming scenarios, Tap Game Customer Service provides an in-game customer service experience. Players don't have to switch apps to use the customer service feature, and they can receive in-game prompts immediately after customer service replies.
```

In addition, the SDK optimizes visuals and interactions for landscape scenarios and provides developers with a control API to reduce the impact of the SDK on the performance of the game's core processes.

### Out-of-the-box development experience

> The Tap Game Customer Service client SDK covers popular game development platforms and provides features such as user interface, login, data reporting, and red dot maintenance. The highly integrated features provide an out-of-the-box development experience.

Reviewer: ```suggestion
The Tap Game Customer Service client SDK covers popular game development platforms and provides features such as user interface, login, data reporting, and badge maintenance. The highly integrated features provide an out-of-the-box development experience.
```

Customer Service supports multiple login methods, including the TDS built-in account feature if the game already uses it, or the game's own account system.

  

> >Still under development
> >Tap Customer Service is currently in public beta, and some features that are still in development may not be available to all users.

Reviewer: ```suggestion
:::info Still under development
Tap Customer Service is currently in public beta, and some features that are still in development may not be available to all users.
:::
```

# TapSupport Introduction

TapSupport assists the game operations team in solving the problems encountered by players faster and better.

## Key Benefits

### Rich Tools to Improve Customer Support Efficiency

TapSupport consists of two core systems: Ticket and Knowledge Base, providing a set of freely combinable functional modules, including announcements, FAQs, forms, etc., to solve players' problems and requests.

- The ticket module comes with the category-based auto-assignment function. You can also customize triggers to achieve a more complex assignment mechanism that can quickly assign tickets to the most appropriate staff members.
    
- Tickets support custom fields that can be filled in by the player or the developer via a form, providing detailed context for the staff.
    
- Multiple notification methods are supported to ensure you don't miss any player responses.
    

### Multi-dimensional Analytics to Support Operations

> TapSupport is designed to help solve player problems, but it is also designed to help the operation team identify issues, refine and accelerate valuable content, and improve the game itself.

Reviewer: ```suggestion
TapSupport is designed to help solve player problems, but it is also designed to help the operation team identify issues, refine valuable content, and improve the game itself.
```

TapSupport provides an intuitive role-based permission control system that supports custom roles in addition to pre-built roles for more flexible needs.

The system provides managers with detailed data to evaluate the quantity and quality of customer support work, including but not limited to the number of tickets handled, the timeliness of responses to players, and player satisfaction.

### User Experience Designed for Players

As a customer support tool focused on gaming scenarios, TapSupport provides an in-game customer support experience. Players don't have to switch apps to use the TapSupport feature, and they can receive in-game prompts immediately after customer support replies.

In addition, the SDK optimizes visuals and interactions for landscape scenarios and provides developers with a control API to reduce the impact of the SDK on the performance of the game's core processes.

### Out-of-the-box Development Experience

The TapSupport client SDK covers popular game development platforms and provides features such as user interface, login, data reporting, and badge maintenance. The highly integrated features provide an out-of-the-box development experience.

> TapSupport multiple login methods, including the TDS built-in account feature if the game already uses it, or the game's own account system.

Reviewer: ```suggestion
TapSupport supports multiple login methods, including the TDS built-in account feature if the game already uses it, or the game's own account system.
```
